UsingLinkClicky.md

```diff
@@ -0,0 +1,396 @@
+# Using Behat
+
+The purpose of testing is to ensure that requirements are met. Additionally regression testing ensures that changes to the code or the site do not make unexpected changes elsewhere. Best practices dictate that we want full testing coverage, yet we want our set of tests DRY (don’t repeat yourself). So, what are the best practices for testing?
+
+Also see [Writing BDD tests for Drupal.org](https://www.drupal.org/node/1578324), [Feature/Scenario Review Checklist](https://www.drupal.org/node/1782858), 
+
+## Analysis
+
+Let’s examine the various considerations of testing our products and sites.
+
+### Functionality
+
+ In our product development we need to test functionality of the site. For our websites this includes:
+
+1. Behaviour: Does the site behave or function as expected?
+
+2. Theme: Does the site have the correct design, layouts, fonts, and colors? This includes menus, block placement, logos, etc.
+
+3. Content: Did our placeholder content and help text get imported properly?
+
+4. Hosting Environment: Where is this site hosted? Is it local on your machine? Is it on sites? On Anchorage? 
+
+5. Authentication Scheme: What approach do use to authenticate? Native Drupal? Webauth? SamL?
+
+### Lifecycle
+
+Testing for our product development and subsequent sites has a sort of timeline or lifecycle. Which looks roughly like this:
+
+    Requirements
+        > Development
```

Comment:
Don't we deploy a website before we develop on it?

Suggested update:

- Reqs
-- Deploy
--- Dev
---- Production
----- Launch
------ Live